docs/make.jl

```diff
@@ -18,15 +18,19 @@ const EXAMPLES_DIR = joinpath(@__DIR__, "..", "examples")
 const OUTPUT_DIR   = joinpath(@__DIR__, "src/literated")
 
 to_be_literated = [
+    #"panantarctic_regional_simulation.jl",
     "single_column_os_papa_simulation.jl",
     "one_degree_simulation.jl",
-    "near_global_ocean_simulation.jl"
+    "near_global_ocean_simulation.jl",
 ]
 
 for file in to_be_literated
     filepath = joinpath(EXAMPLES_DIR, file)
     withenv("JULIA_DEBUG" => "Literate") do
         Literate.markdown(filepath, OUTPUT_DIR; flavor = Literate.DocumenterFlavor(), execute = true)
+        #run(`julia --project=docs -e "using Literate;
+        #    Literate.markdown(\"$filepath\", \"$outputpath\";
+        #                       flavor=Literate.DocumenterFlavor(), execute=true)"`)
```

Comment:
```suggestion
        run(`julia --project=docs -e "using Literate;
            Literate.markdown(\"$filepath\", \"$outputpath\";
                               flavor=Literate.DocumenterFlavor(), execute=true)"`)
```

this step is necessary